src/util.c

```diff
@@ -572,6 +574,18 @@ int fismnt(char *dir)
 	return ismnt("/proc/mounts", dir, NULL);
 }
 
+/* Return 1 if dir is a backed by tmpfs or overlayfs */
+int fistmpfs(char *dir)
+{
+	struct statfs info;
+
+	statfs(dir, &info);
+	if (info.f_type == TMPFS_MAGIC || info.f_type == OVERLAYFS_SUPER_MAGIC)
+		return 1;
```

Comment:
Either initialize the stack variable with `{ 0 }`, and/or check the return value of `statfs()`, e.g., on error `return 0;`